### PR TITLE
add guards for symmetrisation flag to matrixfit and fit.effectivemass

### PR DESCRIPTION
**Pull request by kostrzewa**
Fix errors due to missing ```symmetrised``` member in cf class.

### CODE REVIEW

**Comment by kostrzewa** [comment]
@martin-ueding This should fix the issue that you saw. Sorry about that!

File: R/effectivemass.R [hunk 1]
```diff
@@ -148,7 +148,7 @@ fit.effectivemass <- function(cf, t1, t2, useCov=FALSE, replace.na=TRUE, boot.fi
     stop("t1 and t2 must be specified! Aborting...!\n")
   }
   tmax <- cf$Time/2
-  if(!is.null(cf$cf$symmetrised)) {
+  if( "symmetrised" %in% names(cf) ) {
```
Comment (martin-ueding) [inline R/effectivemass.R]: Before it was `cf$cf$symmetrised`, now it is `names(cf)`. I would think that we need `names(cf$cf)` then.

Comment (kostrzewa) [reply R/effectivemass.R]: Yup.

Comment (kostrzewa) [reply R/effectivemass.R]: Even better would be to not name this object "cf" :)

**Comment by kostrzewa** [comment]
should be fixed now

**Comment by kostrzewa** [comment]
You've probably modified this in the process of updating the cf class. Should I close this then?

**Pull request closed without merging**